CHANGELOG.md

```diff
@@ -7,6 +7,15 @@ This project adheres to the Node [default version scheme](https://docs.npmjs.com
 
 ## [next-version]
 
+### Added
+
+### Changed
+- Internal: Remove unused dependencies and scripts from `package.json`
+
+### Fixed
+- UI: Accessibility - Focus is at document start
+- UI: Camera not detected on Safari 13.1 on iOS 13.4.1, macOS 10.15.4)
```

Comment:
I don't think you need closing `)` here 🙈

Reply:
Oops. You're right! 🙈